flink-staging/flink-gelly/src/main/java/org/apache/flink/graph/Graph.java

```diff
@@ -282,6 +282,58 @@ public void flatMap(Edge<K, EV> edge, Collector<Tuple1<K>> out) {
 	}
 
 	/**
+	* Creates a graph from CSV files.
+	*
+	* Vertices with value are created from a CSV file with 2 fields
+	* Edges with value are created from a CSV file with 3 fields
+	* from Tuple3.
+	*
+	* @param path1 path to a CSV file with the Vertices data.
+	* @param path2 path to a CSV file with the Edges data
+	* @param context the flink execution environment.
+	* @return An instance of {@link org.apache.flink.graph.GraphCsvReader} , which on calling types() method to specify types of the
+	*		 Vertex ID, Vertex Value and Edge value returns a Graph
+	*/
+
+	public static  GraphCsvReader fromCsvReader(String path1, String path2, ExecutionEnvironment context){
+		return (new GraphCsvReader(path1,path2,context));
+	}
+	/** Creates a graph from a CSV file for Edges., Vertices are
+	* induced from the edges.
+	*
+	* Edges with value are created from a CSV file with 3 fields. Vertices are created
+	* automatically and their values are set to NullValue.
+	*
+	* @param path a path to a CSV file with the Edges data
+	* @param context the flink execution environment.
+	* @return An instance of {@link org.apache.flink.graph.GraphCsvReader} , which on calling types() method to specify types of the
+	* Vertex ID, Vertex Value and Edge value returns a Graph
+	*/
+
+	public static GraphCsvReader fromCsvReader(String path, ExecutionEnvironment context){
+		return (new GraphCsvReader(path,context));
```

Comment:
Parentheses here too.
Also, it's nice to have a space after commas when separating arguments and a space before the curly bracket that defines the start of the method.